### ISSUE TITLE
You cannot call a method on a null-valued expression.

### ISSUE THREAD

**Issue by UselessGuru** [MultiPoolMiner/MultiPoolMiner#2441]
```
You cannot call a method on a null-valued expression.
At C:\Users\anon\Downloads\MPM\MultiPoolMiner.ps1:564 char:44
+     $AllPools.Algorithm | ForEach-Object { $_.ToLower() } | Select-Ob ...
+                                            ~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : InvokeMethodOnNull

Select-Object : Property "TotalMinutes" cannot be found.
At C:\Users\anon\Downloads\MPM\MultiPoolMiner.ps1:565 char:386
+ ... m - $_.Minimum } | Select-Object -ExpandProperty TotalMinutes) -gt $S ...
+                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (0:PSObject) [Select-Object], PSArgumentException
+ FullyQualifiedErrorId : ExpandPropertyNotFound,Microsoft.PowerShell.Commands.SelectObjectCommand
```

@tbergeron

Does this happen on every run?

When it happens please provide the data from localhost:3999/config, localhost:3999/newpools and localhost:399/allpools.

_Originally posted by @tbergeron in https://github.com/MultiPoolMiner/MultiPoolMiner/pull/2381#issuecomment-551173749_

**Comment by UselessGuru** [comment]
Will  be fixed with #2453

**Comment by UselessGuru** [comment]
@tbergeron

As it seem you have some broken / non standard pool files in the pool folder.
